docs/metrics/getting-started/Program.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        using var provider = Sdk.CreateMeterProviderBuilder()
+                .AddSource("TestMeter")
+                .AddExportProcessor(new MetricConsoleExporter())
+                .Build();
+
+        using var meter = new Meter("TestMeter", "0.0.1");
```

Comment:
Do we encourage the meter to be static (like [this](https://github.com/open-telemetry/opentelemetry-dotnet/blob/7e40c7b08e4934e1fe10201d7a6aca75d3d8a2ce/docs/trace/getting-started/Program.cs#L23))?

Reply:
I think we should so it similar to Tracing. Not just Meter, even the instrument too.